[2023-05-16 07:07:00 +0000] [32291] [INFO] Handling signal: term
[2023-05-16 07:07:00 +0000] [32294] [INFO] Worker exiting (pid: 32294)
[2023-05-16 07:07:00 +0000] [32293] [INFO] Worker exiting (pid: 32293)
[2023-05-16 07:07:00 +0000] [32295] [INFO] Worker exiting (pid: 32295)
test...................111111111
<Response [200]>
[2023-05-16 07:07:00 +0000] [32291] [INFO] Shutting down: Master
[2023-05-16 07:07:00 +0000] [32433] [INFO] Starting gunicorn 20.1.0
[2023-05-16 07:07:00 +0000] [32433] [INFO] Listening at: http://127.0.0.1:8000 (32433)
[2023-05-16 07:07:00 +0000] [32433] [INFO] Using worker: sync
[2023-05-16 07:07:00 +0000] [32435] [INFO] Booting worker with pid: 32435
[2023-05-16 07:07:00 +0000] [32436] [INFO] Booting worker with pid: 32436
[2023-05-16 07:07:00 +0000] [32437] [INFO] Booting worker with pid: 32437
[2023-05-16 07:08:28 +0000] [32433] [INFO] Handling signal: term
[2023-05-16 07:08:28 +0000] [32437] [INFO] Worker exiting (pid: 32437)
[2023-05-16 07:08:28 +0000] [32435] [INFO] Worker exiting (pid: 32435)
[2023-05-16 07:08:28 +0000] [32436] [INFO] Worker exiting (pid: 32436)
[2023-05-16 07:08:29 +0000] [32433] [INFO] Shutting down: Master
[2023-05-16 07:08:29 +0000] [32521] [INFO] Starting gunicorn 20.1.0
[2023-05-16 07:08:29 +0000] [32521] [INFO] Listening at: http://127.0.0.1:8000 (32521)
[2023-05-16 07:08:29 +0000] [32521] [INFO] Using worker: sync
[2023-05-16 07:08:29 +0000] [32523] [INFO] Booting worker with pid: 32523
[2023-05-16 07:08:29 +0000] [32524] [INFO] Booting worker with pid: 32524
[2023-05-16 07:08:29 +0000] [32525] [INFO] Booting worker with pid: 32525
Not Found: /img/bg.jpg
Not Found: /api/contract_management/contract_customer/787878/detail
Not Found: /api/contract_management/contract_customer/787878/detail
Not Found: /api/contract_management/contract_customer/787878/detail
Internal Server Error: /api/contract_management/787878/available/hospitals
Traceback (most recent call last):
  File "/home/medet/venv/lib/python3.8/site-packages/django/core/handlers/exception.py", line 55, in inner
    response = get_response(request)
  File "/home/medet/venv/lib/python3.8/site-packages/django/core/handlers/base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/home/medet/venv/lib/python3.8/site-packages/django/views/decorators/csrf.py", line 54, in wrapped_view
    return view_func(*args, **kwargs)
  File "/home/medet/venv/lib/python3.8/site-packages/django/views/generic/base.py", line 84, in view
    return self.dispatch(request, *args, **kwargs)
  File "/home/medet/venv/lib/python3.8/site-packages/rest_framework/views.py", line 509, in dispatch
    response = self.handle_exception(exc)
  File "/home/medet/venv/lib/python3.8/site-packages/rest_framework/views.py", line 469, in handle_exception
    self.raise_uncaught_exception(exc)
  File "/home/medet/venv/lib/python3.8/site-packages/rest_framework/views.py", line 480, in raise_uncaught_exception
    raise exc
  File "/home/medet/venv/lib/python3.8/site-packages/rest_framework/views.py", line 506, in dispatch
    response = handler(request, *args, **kwargs)
  File "/home/medet/elmedi/contract_management/api/views.py", line 36, in get
    contract_customer = ContractCustomer.objects.get(number=card_number)
  File "/home/medet/venv/lib/python3.8/site-packages/django/db/models/manager.py", line 85, in manager_method
    return getattr(self.get_queryset(), name)(*args, **kwargs)
  File "/home/medet/venv/lib/python3.8/site-packages/django/db/models/query.py", line 496, in get
    raise self.model.DoesNotExist(
contract_management.models.ContractCustomer.DoesNotExist: ContractCustomer matching query does not exist.
Internal Server Error: /api/contract_management/787878/available/hospitals
Traceback (most recent call last):
  File "/home/medet/venv/lib/python3.8/site-packages/django/core/handlers/exception.py", line 55, in inner
    response = get_response(request)
  File "/home/medet/venv/lib/python3.8/site-packages/django/core/handlers/base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/home/medet/venv/lib/python3.8/site-packages/django/views/decorators/csrf.py", line 54, in wrapped_view
    return view_func(*args, **kwargs)
  File "/home/medet/venv/lib/python3.8/site-packages/django/views/generic/base.py", line 84, in view
    return self.dispatch(request, *args, **kwargs)
  File "/home/medet/venv/lib/python3.8/site-packages/rest_framework/views.py", line 509, in dispatch
    response = self.handle_exception(exc)
  File "/home/medet/venv/lib/python3.8/site-packages/rest_framework/views.py", line 469, in handle_exception
    self.raise_uncaught_exception(exc)
  File "/home/medet/venv/lib/python3.8/site-packages/rest_framework/views.py", line 480, in raise_uncaught_exception
    raise exc
  File "/home/medet/venv/lib/python3.8/site-packages/rest_framework/views.py", line 506, in dispatch
    response = handler(request, *args, **kwargs)
  File "/home/medet/elmedi/contract_management/api/views.py", line 36, in get
    contract_customer = ContractCustomer.objects.get(number=card_number)
  File "/home/medet/venv/lib/python3.8/site-packages/django/db/models/manager.py", line 85, in manager_method
    return getattr(self.get_queryset(), name)(*args, **kwargs)
  File "/home/medet/venv/lib/python3.8/site-packages/django/db/models/query.py", line 496, in get
    raise self.model.DoesNotExist(
contract_management.models.ContractCustomer.DoesNotExist: ContractCustomer matching query does not exist.
Internal Server Error: /api/contract_management/787878/available/hospitals
Traceback (most recent call last):
  File "/home/medet/venv/lib/python3.8/site-packages/django/core/handlers/exception.py", line 55, in inner
    response = get_response(request)
  File "/home/medet/venv/lib/python3.8/site-packages/django/core/handlers/base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/home/medet/venv/lib/python3.8/site-packages/django/views/decorators/csrf.py", line 54, in wrapped_view
    return view_func(*args, **kwargs)
  File "/home/medet/venv/lib/python3.8/site-packages/django/views/generic/base.py", line 84, in view
    return self.dispatch(request, *args, **kwargs)
  File "/home/medet/venv/lib/python3.8/site-packages/rest_framework/views.py", line 509, in dispatch
    response = self.handle_exception(exc)
  File "/home/medet/venv/lib/python3.8/site-packages/rest_framework/views.py", line 469, in handle_exception
    self.raise_uncaught_exception(exc)
  File "/home/medet/venv/lib/python3.8/site-packages/rest_framework/views.py", line 480, in raise_uncaught_exception
    raise exc
  File "/home/medet/venv/lib/python3.8/site-packages/rest_framework/views.py", line 506, in dispatch
    response = handler(request, *args, **kwargs)
  File "/home/medet/elmedi/contract_management/api/views.py", line 36, in get
    contract_customer = ContractCustomer.objects.get(number=card_number)
  File "/home/medet/venv/lib/python3.8/site-packages/django/db/models/manager.py", line 85, in manager_method
    return getattr(self.get_queryset(), name)(*args, **kwargs)
  File "/home/medet/venv/lib/python3.8/site-packages/django/db/models/query.py", line 496, in get
    raise self.model.DoesNotExist(
contract_management.models.ContractCustomer.DoesNotExist: ContractCustomer matching query does not exist.
Internal Server Error: /api/contract_management/787878/available/hospitals
Traceback (most recent call last):
  File "/home/medet/venv/lib/python3.8/site-packages/django/core/handlers/exception.py", line 55, in inner
    response = get_response(request)
  File "/home/medet/venv/lib/python3.8/site-packages/django/core/handlers/base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/home/medet/venv/lib/python3.8/site-packages/django/views/decorators/csrf.py", line 54, in wrapped_view
    return view_func(*args, **kwargs)
  File "/home/medet/venv/lib/python3.8/site-packages/django/views/generic/base.py", line 84, in view
    return self.dispatch(request, *args, **kwargs)
  File "/home/medet/venv/lib/python3.8/site-packages/rest_framework/views.py", line 509, in dispatch
    response = self.handle_exception(exc)
  File "/home/medet/venv/lib/python3.8/site-packages/rest_framework/views.py", line 469, in handle_exception
    self.raise_uncaught_exception(exc)
  File "/home/medet/venv/lib/python3.8/site-packages/rest_framework/views.py", line 480, in raise_uncaught_exception
    raise exc
  File "/home/medet/venv/lib/python3.8/site-packages/rest_framework/views.py", line 506, in dispatch
    response = handler(request, *args, **kwargs)
  File "/home/medet/elmedi/contract_management/api/views.py", line 36, in get
    contract_customer = ContractCustomer.objects.get(number=card_number)
  File "/home/medet/venv/lib/python3.8/site-packages/django/db/models/manager.py", line 85, in manager_method
    return getattr(self.get_queryset(), name)(*args, **kwargs)
  File "/home/medet/venv/lib/python3.8/site-packages/django/db/models/query.py", line 496, in get
    raise self.model.DoesNotExist(
contract_management.models.ContractCustomer.DoesNotExist: ContractCustomer matching query does not exist.
Internal Server Error: /api/contract_management/787878/available/hospitals
Traceback (most recent call last):
  File "/home/medet/venv/lib/python3.8/site-packages/django/core/handlers/exception.py", line 55, in inner
    response = get_response(request)
  File "/home/medet/venv/lib/python3.8/site-packages/django/core/handlers/base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/home/medet/venv/lib/python3.8/site-packages/django/views/decorators/csrf.py", line 54, in wrapped_view
    return view_func(*args, **kwargs)
  File "/home/medet/venv/lib/python3.8/site-packages/django/views/generic/base.py", line 84, in view
    return self.dispatch(request, *args, **kwargs)
  File "/home/medet/venv/lib/python3.8/site-packages/rest_framework/views.py", line 509, in dispatch
    response = self.handle_exception(exc)
  File "/home/medet/venv/lib/python3.8/site-packages/rest_framework/views.py", line 469, in handle_exception
    self.raise_uncaught_exception(exc)
  File "/home/medet/venv/lib/python3.8/site-packages/rest_framework/views.py", line 480, in raise_uncaught_exception
    raise exc
  File "/home/medet/venv/lib/python3.8/site-packages/rest_framework/views.py", line 506, in dispatch
    response = handler(request, *args, **kwargs)
  File "/home/medet/elmedi/contract_management/api/views.py", line 36, in get
    contract_customer = ContractCustomer.objects.get(number=card_number)
  File "/home/medet/venv/lib/python3.8/site-packages/django/db/models/manager.py", line 85, in manager_method
    return getattr(self.get_queryset(), name)(*args, **kwargs)
  File "/home/medet/venv/lib/python3.8/site-packages/django/db/models/query.py", line 496, in get
    raise self.model.DoesNotExist(
contract_management.models.ContractCustomer.DoesNotExist: ContractCustomer matching query does not exist.
Internal Server Error: /api/contract_management/787878/available/hospitals
Traceback (most recent call last):
  File "/home/medet/venv/lib/python3.8/site-packages/django/core/handlers/exception.py", line 55, in inner
    response = get_response(request)
  File "/home/medet/venv/lib/python3.8/site-packages/django/core/handlers/base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/home/medet/venv/lib/python3.8/site-packages/django/views/decorators/csrf.py", line 54, in wrapped_view
    return view_func(*args, **kwargs)
  File "/home/medet/venv/lib/python3.8/site-packages/django/views/generic/base.py", line 84, in view
    return self.dispatch(request, *args, **kwargs)
  File "/home/medet/venv/lib/python3.8/site-packages/rest_framework/views.py", line 509, in dispatch
    response = self.handle_exception(exc)
  File "/home/medet/venv/lib/python3.8/site-packages/rest_framework/views.py", line 469, in handle_exception
    self.raise_uncaught_exception(exc)
  File "/home/medet/venv/lib/python3.8/site-packages/rest_framework/views.py", line 480, in raise_uncaught_exception
    raise exc
  File "/home/medet/venv/lib/python3.8/site-packages/rest_framework/views.py", line 506, in dispatch
    response = handler(request, *args, **kwargs)
  File "/home/medet/elmedi/contract_management/api/views.py", line 36, in get
    contract_customer = ContractCustomer.objects.get(number=card_number)
  File "/home/medet/venv/lib/python3.8/site-packages/django/db/models/manager.py", line 85, in manager_method
    return getattr(self.get_queryset(), name)(*args, **kwargs)
  File "/home/medet/venv/lib/python3.8/site-packages/django/db/models/query.py", line 496, in get
    raise self.model.DoesNotExist(
contract_management.models.ContractCustomer.DoesNotExist: ContractCustomer matching query does not exist.
Internal Server Error: /api/contract_management/787878/available/hospitals
Traceback (most recent call last):
  File "/home/medet/venv/lib/python3.8/site-packages/django/core/handlers/exception.py", line 55, in inner
    response = get_response(request)
  File "/home/medet/venv/lib/python3.8/site-packages/django/core/handlers/base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/home/medet/venv/lib/python3.8/site-packages/django/views/decorators/csrf.py", line 54, in wrapped_view
    return view_func(*args, **kwargs)
  File "/home/medet/venv/lib/python3.8/site-packages/django/views/generic/base.py", line 84, in view
    return self.dispatch(request, *args, **kwargs)
  File "/home/medet/venv/lib/python3.8/site-packages/rest_framework/views.py", line 509, in dispatch
    response = self.handle_exception(exc)
  File "/home/medet/venv/lib/python3.8/site-packages/rest_framework/views.py", line 469, in handle_exception
    self.raise_uncaught_exception(exc)
  File "/home/medet/venv/lib/python3.8/site-packages/rest_framework/views.py", line 480, in raise_uncaught_exception
    raise exc
  File "/home/medet/venv/lib/python3.8/site-packages/rest_framework/views.py", line 506, in dispatch
    response = handler(request, *args, **kwargs)
  File "/home/medet/elmedi/contract_management/api/views.py", line 36, in get
    contract_customer = ContractCustomer.objects.get(number=card_number)
  File "/home/medet/venv/lib/python3.8/site-packages/django/db/models/manager.py", line 85, in manager_method
    return getattr(self.get_queryset(), name)(*args, **kwargs)
  File "/home/medet/venv/lib/python3.8/site-packages/django/db/models/query.py", line 496, in get
    raise self.model.DoesNotExist(
contract_management.models.ContractCustomer.DoesNotExist: ContractCustomer matching query does not exist.
Internal Server Error: /api/contract_management/787878/available/hospitals
Traceback (most recent call last):
  File "/home/medet/venv/lib/python3.8/site-packages/django/core/handlers/exception.py", line 55, in inner
    response = get_response(request)
  File "/home/medet/venv/lib/python3.8/site-packages/django/core/handlers/base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/home/medet/venv/lib/python3.8/site-packages/django/views/decorators/csrf.py", line 54, in wrapped_view
    return view_func(*args, **kwargs)
  File "/home/medet/venv/lib/python3.8/site-packages/django/views/generic/base.py", line 84, in view
    return self.dispatch(request, *args, **kwargs)
  File "/home/medet/venv/lib/python3.8/site-packages/rest_framework/views.py", line 509, in dispatch
    response = self.handle_exception(exc)
  File "/home/medet/venv/lib/python3.8/site-packages/rest_framework/views.py", line 469, in handle_exception
    self.raise_uncaught_exception(exc)
  File "/home/medet/venv/lib/python3.8/site-packages/rest_framework/views.py", line 480, in raise_uncaught_exception
    raise exc
  File "/home/medet/venv/lib/python3.8/site-packages/rest_framework/views.py", line 506, in dispatch
    response = handler(request, *args, **kwargs)
  File "/home/medet/elmedi/contract_management/api/views.py", line 36, in get
    contract_customer = ContractCustomer.objects.get(number=card_number)
  File "/home/medet/venv/lib/python3.8/site-packages/django/db/models/manager.py", line 85, in manager_method
    return getattr(self.get_queryset(), name)(*args, **kwargs)
  File "/home/medet/venv/lib/python3.8/site-packages/django/db/models/query.py", line 496, in get
    raise self.model.DoesNotExist(
contract_management.models.ContractCustomer.DoesNotExist: ContractCustomer matching query does not exist.
Internal Server Error: /api/contract_management/787878/available/hospitals
Traceback (most recent call last):
  File "/home/medet/venv/lib/python3.8/site-packages/django/core/handlers/exception.py", line 55, in inner
    response = get_response(request)
  File "/home/medet/venv/lib/python3.8/site-packages/django/core/handlers/base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/home/medet/venv/lib/python3.8/site-packages/django/views/decorators/csrf.py", line 54, in wrapped_view
    return view_func(*args, **kwargs)
  File "/home/medet/venv/lib/python3.8/site-packages/django/views/generic/base.py", line 84, in view
    return self.dispatch(request, *args, **kwargs)
  File "/home/medet/venv/lib/python3.8/site-packages/rest_framework/views.py", line 509, in dispatch
    response = self.handle_exception(exc)
  File "/home/medet/venv/lib/python3.8/site-packages/rest_framework/views.py", line 469, in handle_exception
    self.raise_uncaught_exception(exc)
  File "/home/medet/venv/lib/python3.8/site-packages/rest_framework/views.py", line 480, in raise_uncaught_exception
    raise exc
  File "/home/medet/venv/lib/python3.8/site-packages/rest_framework/views.py", line 506, in dispatch
    response = handler(request, *args, **kwargs)
  File "/home/medet/elmedi/contract_management/api/views.py", line 36, in get
    contract_customer = ContractCustomer.objects.get(number=card_number)
  File "/home/medet/venv/lib/python3.8/site-packages/django/db/models/manager.py", line 85, in manager_method
    return getattr(self.get_queryset(), name)(*args, **kwargs)
  File "/home/medet/venv/lib/python3.8/site-packages/django/db/models/query.py", line 496, in get
    raise self.model.DoesNotExist(
contract_management.models.ContractCustomer.DoesNotExist: ContractCustomer matching query does not exist.
Internal Server Error: /api/contract_management/787878/available/hospitals
Traceback (most recent call last):
  File "/home/medet/venv/lib/python3.8/site-packages/django/core/handlers/exception.py", line 55, in inner
    response = get_response(request)
  File "/home/medet/venv/lib/python3.8/site-packages/django/core/handlers/base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/home/medet/venv/lib/python3.8/site-packages/django/views/decorators/csrf.py", line 54, in wrapped_view
    return view_func(*args, **kwargs)
  File "/home/medet/venv/lib/python3.8/site-packages/django/views/generic/base.py", line 84, in view
    return self.dispatch(request, *args, **kwargs)
  File "/home/medet/venv/lib/python3.8/site-packages/rest_framework/views.py", line 509, in dispatch
    response = self.handle_exception(exc)
  File "/home/medet/venv/lib/python3.8/site-packages/rest_framework/views.py", line 469, in handle_exception
    self.raise_uncaught_exception(exc)
  File "/home/medet/venv/lib/python3.8/site-packages/rest_framework/views.py", line 480, in raise_uncaught_exception
    raise exc
  File "/home/medet/venv/lib/python3.8/site-packages/rest_framework/views.py", line 506, in dispatch
    response = handler(request, *args, **kwargs)
  File "/home/medet/elmedi/contract_management/api/views.py", line 36, in get
    contract_customer = ContractCustomer.objects.get(number=card_number)
  File "/home/medet/venv/lib/python3.8/site-packages/django/db/models/manager.py", line 85, in manager_method
    return getattr(self.get_queryset(), name)(*args, **kwargs)
  File "/home/medet/venv/lib/python3.8/site-packages/django/db/models/query.py", line 496, in get
    raise self.model.DoesNotExist(
contract_management.models.ContractCustomer.DoesNotExist: ContractCustomer matching query does not exist.
Internal Server Error: /api/contract_management/787878/available/hospitals
Traceback (most recent call last):
  File "/home/medet/venv/lib/python3.8/site-packages/django/core/handlers/exception.py", line 55, in inner
    response = get_response(request)
  File "/home/medet/venv/lib/python3.8/site-packages/django/core/handlers/base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/home/medet/venv/lib/python3.8/site-packages/django/views/decorators/csrf.py", line 54, in wrapped_view
    return view_func(*args, **kwargs)
  File "/home/medet/venv/lib/python3.8/site-packages/django/views/generic/base.py", line 84, in view
    return self.dispatch(request, *args, **kwargs)
  File "/home/medet/venv/lib/python3.8/site-packages/rest_framework/views.py", line 509, in dispatch
    response = self.handle_exception(exc)
  File "/home/medet/venv/lib/python3.8/site-packages/rest_framework/views.py", line 469, in handle_exception
    self.raise_uncaught_exception(exc)
  File "/home/medet/venv/lib/python3.8/site-packages/rest_framework/views.py", line 480, in raise_uncaught_exception
    raise exc
  File "/home/medet/venv/lib/python3.8/site-packages/rest_framework/views.py", line 506, in dispatch
    response = handler(request, *args, **kwargs)
  File "/home/medet/elmedi/contract_management/api/views.py", line 36, in get
    contract_customer = ContractCustomer.objects.get(number=card_number)
  File "/home/medet/venv/lib/python3.8/site-packages/django/db/models/manager.py", line 85, in manager_method
    return getattr(self.get_queryset(), name)(*args, **kwargs)
  File "/home/medet/venv/lib/python3.8/site-packages/django/db/models/query.py", line 496, in get
    raise self.model.DoesNotExist(
contract_management.models.ContractCustomer.DoesNotExist: ContractCustomer matching query does not exist.
Internal Server Error: /api/contract_management/787878/available/hospitals
Traceback (most recent call last):
  File "/home/medet/venv/lib/python3.8/site-packages/django/core/handlers/exception.py", line 55, in inner
    response = get_response(request)
  File "/home/medet/venv/lib/python3.8/site-packages/django/core/handlers/base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/home/medet/venv/lib/python3.8/site-packages/django/views/decorators/csrf.py", line 54, in wrapped_view
    return view_func(*args, **kwargs)
  File "/home/medet/venv/lib/python3.8/site-packages/django/views/generic/base.py", line 84, in view
    return self.dispatch(request, *args, **kwargs)
  File "/home/medet/venv/lib/python3.8/site-packages/rest_framework/views.py", line 509, in dispatch
    response = self.handle_exception(exc)
  File "/home/medet/venv/lib/python3.8/site-packages/rest_framework/views.py", line 469, in handle_exception
    self.raise_uncaught_exception(exc)
  File "/home/medet/venv/lib/python3.8/site-packages/rest_framework/views.py", line 480, in raise_uncaught_exception
    raise exc
  File "/home/medet/venv/lib/python3.8/site-packages/rest_framework/views.py", line 506, in dispatch
    response = handler(request, *args, **kwargs)
  File "/home/medet/elmedi/contract_management/api/views.py", line 36, in get
    contract_customer = ContractCustomer.objects.get(number=card_number)
  File "/home/medet/venv/lib/python3.8/site-packages/django/db/models/manager.py", line 85, in manager_method
    return getattr(self.get_queryset(), name)(*args, **kwargs)
  File "/home/medet/venv/lib/python3.8/site-packages/django/db/models/query.py", line 496, in get
    raise self.model.DoesNotExist(
contract_management.models.ContractCustomer.DoesNotExist: ContractCustomer matching query does not exist.
Internal Server Error: /api/contract_management/787878/available/hospitals
Traceback (most recent call last):
  File "/home/medet/venv/lib/python3.8/site-packages/django/core/handlers/exception.py", line 55, in inner
    response = get_response(request)
  File "/home/medet/venv/lib/python3.8/site-packages/django/core/handlers/base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/home/medet/venv/lib/python3.8/site-packages/django/views/decorators/csrf.py", line 54, in wrapped_view
    return view_func(*args, **kwargs)
  File "/home/medet/venv/lib/python3.8/site-packages/django/views/generic/base.py", line 84, in view
    return self.dispatch(request, *args, **kwargs)
  File "/home/medet/venv/lib/python3.8/site-packages/rest_framework/views.py", line 509, in dispatch
    response = self.handle_exception(exc)
  File "/home/medet/venv/lib/python3.8/site-packages/rest_framework/views.py", line 469, in handle_exception
    self.raise_uncaught_exception(exc)
  File "/home/medet/venv/lib/python3.8/site-packages/rest_framework/views.py", line 480, in raise_uncaught_exception
    raise exc
  File "/home/medet/venv/lib/python3.8/site-packages/rest_framework/views.py", line 506, in dispatch
    response = handler(request, *args, **kwargs)
  File "/home/medet/elmedi/contract_management/api/views.py", line 36, in get
    contract_customer = ContractCustomer.objects.get(number=card_number)
  File "/home/medet/venv/lib/python3.8/site-packages/django/db/models/manager.py", line 85, in manager_method
    return getattr(self.get_queryset(), name)(*args, **kwargs)
  File "/home/medet/venv/lib/python3.8/site-packages/django/db/models/query.py", line 496, in get
    raise self.model.DoesNotExist(
contract_management.models.ContractCustomer.DoesNotExist: ContractCustomer matching query does not exist.
Internal Server Error: /api/contract_management/787878/available/hospitals
Traceback (most recent call last):
  File "/home/medet/venv/lib/python3.8/site-packages/django/core/handlers/exception.py", line 55, in inner
    response = get_response(request)
  File "/home/medet/venv/lib/python3.8/site-packages/django/core/handlers/base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/home/medet/venv/lib/python3.8/site-packages/django/views/decorators/csrf.py", line 54, in wrapped_view
    return view_func(*args, **kwargs)
  File "/home/medet/venv/lib/python3.8/site-packages/django/views/generic/base.py", line 84, in view
    return self.dispatch(request, *args, **kwargs)
  File "/home/medet/venv/lib/python3.8/site-packages/rest_framework/views.py", line 509, in dispatch
    response = self.handle_exception(exc)
  File "/home/medet/venv/lib/python3.8/site-packages/rest_framework/views.py", line 469, in handle_exception
    self.raise_uncaught_exception(exc)
  File "/home/medet/venv/lib/python3.8/site-packages/rest_framework/views.py", line 480, in raise_uncaught_exception
    raise exc
  File "/home/medet/venv/lib/python3.8/site-packages/rest_framework/views.py", line 506, in dispatch
    response = handler(request, *args, **kwargs)
  File "/home/medet/elmedi/contract_management/api/views.py", line 36, in get
    contract_customer = ContractCustomer.objects.get(number=card_number)
  File "/home/medet/venv/lib/python3.8/site-packages/django/db/models/manager.py", line 85, in manager_method
    return getattr(self.get_queryset(), name)(*args, **kwargs)
  File "/home/medet/venv/lib/python3.8/site-packages/django/db/models/query.py", line 496, in get
    raise self.model.DoesNotExist(
contract_management.models.ContractCustomer.DoesNotExist: ContractCustomer matching query does not exist.
Not Found: /
Not Found: /
Internal Server Error: /api/contract_management/787878/available/hospitals
Traceback (most recent call last):
  File "/home/medet/venv/lib/python3.8/site-packages/django/core/handlers/exception.py", line 55, in inner
    response = get_response(request)
  File "/home/medet/venv/lib/python3.8/site-packages/django/core/handlers/base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/home/medet/venv/lib/python3.8/site-packages/django/views/decorators/csrf.py", line 54, in wrapped_view
    return view_func(*args, **kwargs)
  File "/home/medet/venv/lib/python3.8/site-packages/django/views/generic/base.py", line 84, in view
    return self.dispatch(request, *args, **kwargs)
  File "/home/medet/venv/lib/python3.8/site-packages/rest_framework/views.py", line 509, in dispatch
    response = self.handle_exception(exc)
  File "/home/medet/venv/lib/python3.8/site-packages/rest_framework/views.py", line 469, in handle_exception
    self.raise_uncaught_exception(exc)
  File "/home/medet/venv/lib/python3.8/site-packages/rest_framework/views.py", line 480, in raise_uncaught_exception
    raise exc
  File "/home/medet/venv/lib/python3.8/site-packages/rest_framework/views.py", line 506, in dispatch
    response = handler(request, *args, **kwargs)
  File "/home/medet/elmedi/contract_management/api/views.py", line 36, in get
    contract_customer = ContractCustomer.objects.get(number=card_number)
  File "/home/medet/venv/lib/python3.8/site-packages/django/db/models/manager.py", line 85, in manager_method
    return getattr(self.get_queryset(), name)(*args, **kwargs)
  File "/home/medet/venv/lib/python3.8/site-packages/django/db/models/query.py", line 496, in get
    raise self.model.DoesNotExist(
contract_management.models.ContractCustomer.DoesNotExist: ContractCustomer matching query does not exist.
Internal Server Error: /api/contract_management/787878/available/hospitals
Traceback (most recent call last):
  File "/home/medet/venv/lib/python3.8/site-packages/django/core/handlers/exception.py", line 55, in inner
    response = get_response(request)
  File "/home/medet/venv/lib/python3.8/site-packages/django/core/handlers/base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/home/medet/venv/lib/python3.8/site-packages/django/views/decorators/csrf.py", line 54, in wrapped_view
    return view_func(*args, **kwargs)
  File "/home/medet/venv/lib/python3.8/site-packages/django/views/generic/base.py", line 84, in view
    return self.dispatch(request, *args, **kwargs)
  File "/home/medet/venv/lib/python3.8/site-packages/rest_framework/views.py", line 509, in dispatch
    response = self.handle_exception(exc)
  File "/home/medet/venv/lib/python3.8/site-packages/rest_framework/views.py", line 469, in handle_exception
    self.raise_uncaught_exception(exc)
  File "/home/medet/venv/lib/python3.8/site-packages/rest_framework/views.py", line 480, in raise_uncaught_exception
    raise exc
  File "/home/medet/venv/lib/python3.8/site-packages/rest_framework/views.py", line 506, in dispatch
    response = handler(request, *args, **kwargs)
  File "/home/medet/elmedi/contract_management/api/views.py", line 36, in get
    contract_customer = ContractCustomer.objects.get(number=card_number)
  File "/home/medet/venv/lib/python3.8/site-packages/django/db/models/manager.py", line 85, in manager_method
    return getattr(self.get_queryset(), name)(*args, **kwargs)
  File "/home/medet/venv/lib/python3.8/site-packages/django/db/models/query.py", line 496, in get
    raise self.model.DoesNotExist(
contract_management.models.ContractCustomer.DoesNotExist: ContractCustomer matching query does not exist.
Internal Server Error: /api/contract_management/787878/available/hospitals
Traceback (most recent call last):
  File "/home/medet/venv/lib/python3.8/site-packages/django/core/handlers/exception.py", line 55, in inner
    response = get_response(request)
  File "/home/medet/venv/lib/python3.8/site-packages/django/core/handlers/base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/home/medet/venv/lib/python3.8/site-packages/django/views/decorators/csrf.py", line 54, in wrapped_view
    return view_func(*args, **kwargs)
  File "/home/medet/venv/lib/python3.8/site-packages/django/views/generic/base.py", line 84, in view
    return self.dispatch(request, *args, **kwargs)
  File "/home/medet/venv/lib/python3.8/site-packages/rest_framework/views.py", line 509, in dispatch
    response = self.handle_exception(exc)
  File "/home/medet/venv/lib/python3.8/site-packages/rest_framework/views.py", line 469, in handle_exception
    self.raise_uncaught_exception(exc)
  File "/home/medet/venv/lib/python3.8/site-packages/rest_framework/views.py", line 480, in raise_uncaught_exception
    raise exc
  File "/home/medet/venv/lib/python3.8/site-packages/rest_framework/views.py", line 506, in dispatch
    response = handler(request, *args, **kwargs)
  File "/home/medet/elmedi/contract_management/api/views.py", line 36, in get
    contract_customer = ContractCustomer.objects.get(number=card_number)
  File "/home/medet/venv/lib/python3.8/site-packages/django/db/models/manager.py", line 85, in manager_method
    return getattr(self.get_queryset(), name)(*args, **kwargs)
  File "/home/medet/venv/lib/python3.8/site-packages/django/db/models/query.py", line 496, in get
    raise self.model.DoesNotExist(
contract_management.models.ContractCustomer.DoesNotExist: ContractCustomer matching query does not exist.
Internal Server Error: /api/contract_management/787878/available/hospitals
Traceback (most recent call last):
  File "/home/medet/venv/lib/python3.8/site-packages/django/core/handlers/exception.py", line 55, in inner
    response = get_response(request)
  File "/home/medet/venv/lib/python3.8/site-packages/django/core/handlers/base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/home/medet/venv/lib/python3.8/site-packages/django/views/decorators/csrf.py", line 54, in wrapped_view
    return view_func(*args, **kwargs)
  File "/home/medet/venv/lib/python3.8/site-packages/django/views/generic/base.py", line 84, in view
    return self.dispatch(request, *args, **kwargs)
  File "/home/medet/venv/lib/python3.8/site-packages/rest_framework/views.py", line 509, in dispatch
    response = self.handle_exception(exc)
  File "/home/medet/venv/lib/python3.8/site-packages/rest_framework/views.py", line 469, in handle_exception
    self.raise_uncaught_exception(exc)
  File "/home/medet/venv/lib/python3.8/site-packages/rest_framework/views.py", line 480, in raise_uncaught_exception
    raise exc
  File "/home/medet/venv/lib/python3.8/site-packages/rest_framework/views.py", line 506, in dispatch
    response = handler(request, *args, **kwargs)
  File "/home/medet/elmedi/contract_management/api/views.py", line 36, in get
    contract_customer = ContractCustomer.objects.get(number=card_number)
  File "/home/medet/venv/lib/python3.8/site-packages/django/db/models/manager.py", line 85, in manager_method
    return getattr(self.get_queryset(), name)(*args, **kwargs)
  File "/home/medet/venv/lib/python3.8/site-packages/django/db/models/query.py", line 496, in get
    raise self.model.DoesNotExist(
contract_management.models.ContractCustomer.DoesNotExist: ContractCustomer matching query does not exist.
Internal Server Error: /api/contract_management/787878/available/hospitals
Traceback (most recent call last):
  File "/home/medet/venv/lib/python3.8/site-packages/django/core/handlers/exception.py", line 55, in inner
    response = get_response(request)
  File "/home/medet/venv/lib/python3.8/site-packages/django/core/handlers/base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/home/medet/venv/lib/python3.8/site-packages/django/views/decorators/csrf.py", line 54, in wrapped_view
    return view_func(*args, **kwargs)
  File "/home/medet/venv/lib/python3.8/site-packages/django/views/generic/base.py", line 84, in view
    return self.dispatch(request, *args, **kwargs)
  File "/home/medet/venv/lib/python3.8/site-packages/rest_framework/views.py", line 509, in dispatch
    response = self.handle_exception(exc)
  File "/home/medet/venv/lib/python3.8/site-packages/rest_framework/views.py", line 469, in handle_exception
    self.raise_uncaught_exception(exc)
  File "/home/medet/venv/lib/python3.8/site-packages/rest_framework/views.py", line 480, in raise_uncaught_exception
    raise exc
  File "/home/medet/venv/lib/python3.8/site-packages/rest_framework/views.py", line 506, in dispatch
    response = handler(request, *args, **kwargs)
  File "/home/medet/elmedi/contract_management/api/views.py", line 36, in get
    contract_customer = ContractCustomer.objects.get(number=card_number)
  File "/home/medet/venv/lib/python3.8/site-packages/django/db/models/manager.py", line 85, in manager_method
    return getattr(self.get_queryset(), name)(*args, **kwargs)
  File "/home/medet/venv/lib/python3.8/site-packages/django/db/models/query.py", line 496, in get
    raise self.model.DoesNotExist(
contract_management.models.ContractCustomer.DoesNotExist: ContractCustomer matching query does not exist.
Internal Server Error: /api/contract_management/787878/available/hospitals
Traceback (most recent call last):
  File "/home/medet/venv/lib/python3.8/site-packages/django/core/handlers/exception.py", line 55, in inner
    response = get_response(request)
  File "/home/medet/venv/lib/python3.8/site-packages/django/core/handlers/base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/home/medet/venv/lib/python3.8/site-packages/django/views/decorators/csrf.py", line 54, in wrapped_view
    return view_func(*args, **kwargs)
  File "/home/medet/venv/lib/python3.8/site-packages/django/views/generic/base.py", line 84, in view
    return self.dispatch(request, *args, **kwargs)
  File "/home/medet/venv/lib/python3.8/site-packages/rest_framework/views.py", line 509, in dispatch
    response = self.handle_exception(exc)
  File "/home/medet/venv/lib/python3.8/site-packages/rest_framework/views.py", line 469, in handle_exception
    self.raise_uncaught_exception(exc)
  File "/home/medet/venv/lib/python3.8/site-packages/rest_framework/views.py", line 480, in raise_uncaught_exception
    raise exc
  File "/home/medet/venv/lib/python3.8/site-packages/rest_framework/views.py", line 506, in dispatch
    response = handler(request, *args, **kwargs)
  File "/home/medet/elmedi/contract_management/api/views.py", line 36, in get
    contract_customer = ContractCustomer.objects.get(number=card_number)
  File "/home/medet/venv/lib/python3.8/site-packages/django/db/models/manager.py", line 85, in manager_method
    return getattr(self.get_queryset(), name)(*args, **kwargs)
  File "/home/medet/venv/lib/python3.8/site-packages/django/db/models/query.py", line 496, in get
    raise self.model.DoesNotExist(
contract_management.models.ContractCustomer.DoesNotExist: ContractCustomer matching query does not exist.
Internal Server Error: /api/contract_management/787878/available/hospitals
Traceback (most recent call last):
  File "/home/medet/venv/lib/python3.8/site-packages/django/core/handlers/exception.py", line 55, in inner
    response = get_response(request)
  File "/home/medet/venv/lib/python3.8/site-packages/django/core/handlers/base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/home/medet/venv/lib/python3.8/site-packages/django/views/decorators/csrf.py", line 54, in wrapped_view
    return view_func(*args, **kwargs)
  File "/home/medet/venv/lib/python3.8/site-packages/django/views/generic/base.py", line 84, in view
    return self.dispatch(request, *args, **kwargs)
  File "/home/medet/venv/lib/python3.8/site-packages/rest_framework/views.py", line 509, in dispatch
    response = self.handle_exception(exc)
  File "/home/medet/venv/lib/python3.8/site-packages/rest_framework/views.py", line 469, in handle_exception
    self.raise_uncaught_exception(exc)
  File "/home/medet/venv/lib/python3.8/site-packages/rest_framework/views.py", line 480, in raise_uncaught_exception
    raise exc
  File "/home/medet/venv/lib/python3.8/site-packages/rest_framework/views.py", line 506, in dispatch
    response = handler(request, *args, **kwargs)
  File "/home/medet/elmedi/contract_management/api/views.py", line 36, in get
    contract_customer = ContractCustomer.objects.get(number=card_number)
  File "/home/medet/venv/lib/python3.8/site-packages/django/db/models/manager.py", line 85, in manager_method
    return getattr(self.get_queryset(), name)(*args, **kwargs)
  File "/home/medet/venv/lib/python3.8/site-packages/django/db/models/query.py", line 496, in get
    raise self.model.DoesNotExist(
contract_management.models.ContractCustomer.DoesNotExist: ContractCustomer matching query does not exist.
Internal Server Error: /api/contract_management/787878/available/hospitals
Traceback (most recent call last):
  File "/home/medet/venv/lib/python3.8/site-packages/django/core/handlers/exception.py", line 55, in inner
    response = get_response(request)
  File "/home/medet/venv/lib/python3.8/site-packages/django/core/handlers/base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/home/medet/venv/lib/python3.8/site-packages/django/views/decorators/csrf.py", line 54, in wrapped_view
    return view_func(*args, **kwargs)
  File "/home/medet/venv/lib/python3.8/site-packages/django/views/generic/base.py", line 84, in view
    return self.dispatch(request, *args, **kwargs)
  File "/home/medet/venv/lib/python3.8/site-packages/rest_framework/views.py", line 509, in dispatch
    response = self.handle_exception(exc)
  File "/home/medet/venv/lib/python3.8/site-packages/rest_framework/views.py", line 469, in handle_exception
    self.raise_uncaught_exception(exc)
  File "/home/medet/venv/lib/python3.8/site-packages/rest_framework/views.py", line 480, in raise_uncaught_exception
    raise exc
  File "/home/medet/venv/lib/python3.8/site-packages/rest_framework/views.py", line 506, in dispatch
    response = handler(request, *args, **kwargs)
  File "/home/medet/elmedi/contract_management/api/views.py", line 36, in get
    contract_customer = ContractCustomer.objects.get(number=card_number)
  File "/home/medet/venv/lib/python3.8/site-packages/django/db/models/manager.py", line 85, in manager_method
    return getattr(self.get_queryset(), name)(*args, **kwargs)
  File "/home/medet/venv/lib/python3.8/site-packages/django/db/models/query.py", line 496, in get
    raise self.model.DoesNotExist(
contract_management.models.ContractCustomer.DoesNotExist: ContractCustomer matching query does not exist.
Internal Server Error: /api/contract_management/787878/available/hospitals
Traceback (most recent call last):
  File "/home/medet/venv/lib/python3.8/site-packages/django/core/handlers/exception.py", line 55, in inner
    response = get_response(request)
  File "/home/medet/venv/lib/python3.8/site-packages/django/core/handlers/base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/home/medet/venv/lib/python3.8/site-packages/django/views/decorators/csrf.py", line 54, in wrapped_view
    return view_func(*args, **kwargs)
  File "/home/medet/venv/lib/python3.8/site-packages/django/views/generic/base.py", line 84, in view
    return self.dispatch(request, *args, **kwargs)
  File "/home/medet/venv/lib/python3.8/site-packages/rest_framework/views.py", line 509, in dispatch
    response = self.handle_exception(exc)
  File "/home/medet/venv/lib/python3.8/site-packages/rest_framework/views.py", line 469, in handle_exception
    self.raise_uncaught_exception(exc)
  File "/home/medet/venv/lib/python3.8/site-packages/rest_framework/views.py", line 480, in raise_uncaught_exception
    raise exc
  File "/home/medet/venv/lib/python3.8/site-packages/rest_framework/views.py", line 506, in dispatch
    response = handler(request, *args, **kwargs)
  File "/home/medet/elmedi/contract_management/api/views.py", line 36, in get
    contract_customer = ContractCustomer.objects.get(number=card_number)
  File "/home/medet/venv/lib/python3.8/site-packages/django/db/models/manager.py", line 85, in manager_method
    return getattr(self.get_queryset(), name)(*args, **kwargs)
  File "/home/medet/venv/lib/python3.8/site-packages/django/db/models/query.py", line 496, in get
    raise self.model.DoesNotExist(
contract_management.models.ContractCustomer.DoesNotExist: ContractCustomer matching query does not exist.
Internal Server Error: /api/contract_management/787878/available/hospitals
Traceback (most recent call last):
  File "/home/medet/venv/lib/python3.8/site-packages/django/core/handlers/exception.py", line 55, in inner
    response = get_response(request)
  File "/home/medet/venv/lib/python3.8/site-packages/django/core/handlers/base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/home/medet/venv/lib/python3.8/site-packages/django/views/decorators/csrf.py", line 54, in wrapped_view
    return view_func(*args, **kwargs)
  File "/home/medet/venv/lib/python3.8/site-packages/django/views/generic/base.py", line 84, in view
    return self.dispatch(request, *args, **kwargs)
  File "/home/medet/venv/lib/python3.8/site-packages/rest_framework/views.py", line 509, in dispatch
    response = self.handle_exception(exc)
  File "/home/medet/venv/lib/python3.8/site-packages/rest_framework/views.py", line 469, in handle_exception
    self.raise_uncaught_exception(exc)
  File "/home/medet/venv/lib/python3.8/site-packages/rest_framework/views.py", line 480, in raise_uncaught_exception
    raise exc
  File "/home/medet/venv/lib/python3.8/site-packages/rest_framework/views.py", line 506, in dispatch
    response = handler(request, *args, **kwargs)
  File "/home/medet/elmedi/contract_management/api/views.py", line 36, in get
    contract_customer = ContractCustomer.objects.get(number=card_number)
  File "/home/medet/venv/lib/python3.8/site-packages/django/db/models/manager.py", line 85, in manager_method
    return getattr(self.get_queryset(), name)(*args, **kwargs)
  File "/home/medet/venv/lib/python3.8/site-packages/django/db/models/query.py", line 496, in get
    raise self.model.DoesNotExist(
contract_management.models.ContractCustomer.DoesNotExist: ContractCustomer matching query does not exist.
Internal Server Error: /api/contract_management/787878/available/hospitals
Traceback (most recent call last):
  File "/home/medet/venv/lib/python3.8/site-packages/django/core/handlers/exception.py", line 55, in inner
    response = get_response(request)
  File "/home/medet/venv/lib/python3.8/site-packages/django/core/handlers/base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/home/medet/venv/lib/python3.8/site-packages/django/views/decorators/csrf.py", line 54, in wrapped_view
    return view_func(*args, **kwargs)
  File "/home/medet/venv/lib/python3.8/site-packages/django/views/generic/base.py", line 84, in view
    return self.dispatch(request, *args, **kwargs)
  File "/home/medet/venv/lib/python3.8/site-packages/rest_framework/views.py", line 509, in dispatch
    response = self.handle_exception(exc)
  File "/home/medet/venv/lib/python3.8/site-packages/rest_framework/views.py", line 469, in handle_exception
    self.raise_uncaught_exception(exc)
  File "/home/medet/venv/lib/python3.8/site-packages/rest_framework/views.py", line 480, in raise_uncaught_exception
    raise exc
  File "/home/medet/venv/lib/python3.8/site-packages/rest_framework/views.py", line 506, in dispatch
    response = handler(request, *args, **kwargs)
  File "/home/medet/elmedi/contract_management/api/views.py", line 36, in get
    contract_customer = ContractCustomer.objects.get(number=card_number)
  File "/home/medet/venv/lib/python3.8/site-packages/django/db/models/manager.py", line 85, in manager_method
    return getattr(self.get_queryset(), name)(*args, **kwargs)
  File "/home/medet/venv/lib/python3.8/site-packages/django/db/models/query.py", line 496, in get
    raise self.model.DoesNotExist(
contract_management.models.ContractCustomer.DoesNotExist: ContractCustomer matching query does not exist.
Internal Server Error: /api/contract_management/787878/available/hospitals
Traceback (most recent call last):
  File "/home/medet/venv/lib/python3.8/site-packages/django/core/handlers/exception.py", line 55, in inner
    response = get_response(request)
  File "/home/medet/venv/lib/python3.8/site-packages/django/core/handlers/base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/home/medet/venv/lib/python3.8/site-packages/django/views/decorators/csrf.py", line 54, in wrapped_view
    return view_func(*args, **kwargs)
  File "/home/medet/venv/lib/python3.8/site-packages/django/views/generic/base.py", line 84, in view
    return self.dispatch(request, *args, **kwargs)
  File "/home/medet/venv/lib/python3.8/site-packages/rest_framework/views.py", line 509, in dispatch
    response = self.handle_exception(exc)
  File "/home/medet/venv/lib/python3.8/site-packages/rest_framework/views.py", line 469, in handle_exception
    self.raise_uncaught_exception(exc)
  File "/home/medet/venv/lib/python3.8/site-packages/rest_framework/views.py", line 480, in raise_uncaught_exception
    raise exc
  File "/home/medet/venv/lib/python3.8/site-packages/rest_framework/views.py", line 506, in dispatch
    response = handler(request, *args, **kwargs)
  File "/home/medet/elmedi/contract_management/api/views.py", line 36, in get
    contract_customer = ContractCustomer.objects.get(number=card_number)
  File "/home/medet/venv/lib/python3.8/site-packages/django/db/models/manager.py", line 85, in manager_method
    return getattr(self.get_queryset(), name)(*args, **kwargs)
  File "/home/medet/venv/lib/python3.8/site-packages/django/db/models/query.py", line 496, in get
    raise self.model.DoesNotExist(
contract_management.models.ContractCustomer.DoesNotExist: ContractCustomer matching query does not exist.
Internal Server Error: /api/contract_management/787878/available/hospitals
Traceback (most recent call last):
  File "/home/medet/venv/lib/python3.8/site-packages/django/core/handlers/exception.py", line 55, in inner
    response = get_response(request)
  File "/home/medet/venv/lib/python3.8/site-packages/django/core/handlers/base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/home/medet/venv/lib/python3.8/site-packages/django/views/decorators/csrf.py", line 54, in wrapped_view
    return view_func(*args, **kwargs)
  File "/home/medet/venv/lib/python3.8/site-packages/django/views/generic/base.py", line 84, in view
    return self.dispatch(request, *args, **kwargs)
  File "/home/medet/venv/lib/python3.8/site-packages/rest_framework/views.py", line 509, in dispatch
    response = self.handle_exception(exc)
  File "/home/medet/venv/lib/python3.8/site-packages/rest_framework/views.py", line 469, in handle_exception
    self.raise_uncaught_exception(exc)
  File "/home/medet/venv/lib/python3.8/site-packages/rest_framework/views.py", line 480, in raise_uncaught_exception
    raise exc
  File "/home/medet/venv/lib/python3.8/site-packages/rest_framework/views.py", line 506, in dispatch
    response = handler(request, *args, **kwargs)
  File "/home/medet/elmedi/contract_management/api/views.py", line 36, in get
    contract_customer = ContractCustomer.objects.get(number=card_number)
  File "/home/medet/venv/lib/python3.8/site-packages/django/db/models/manager.py", line 85, in manager_method
    return getattr(self.get_queryset(), name)(*args, **kwargs)
  File "/home/medet/venv/lib/python3.8/site-packages/django/db/models/query.py", line 496, in get
    raise self.model.DoesNotExist(
contract_management.models.ContractCustomer.DoesNotExist: ContractCustomer matching query does not exist.
Internal Server Error: /api/contract_management/787878/available/hospitals
Traceback (most recent call last):
  File "/home/medet/venv/lib/python3.8/site-packages/django/core/handlers/exception.py", line 55, in inner
    response = get_response(request)
  File "/home/medet/venv/lib/python3.8/site-packages/django/core/handlers/base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/home/medet/venv/lib/python3.8/site-packages/django/views/decorators/csrf.py", line 54, in wrapped_view
    return view_func(*args, **kwargs)
  File "/home/medet/venv/lib/python3.8/site-packages/django/views/generic/base.py", line 84, in view
    return self.dispatch(request, *args, **kwargs)
  File "/home/medet/venv/lib/python3.8/site-packages/rest_framework/views.py", line 509, in dispatch
    response = self.handle_exception(exc)
  File "/home/medet/venv/lib/python3.8/site-packages/rest_framework/views.py", line 469, in handle_exception
    self.raise_uncaught_exception(exc)
  File "/home/medet/venv/lib/python3.8/site-packages/rest_framework/views.py", line 480, in raise_uncaught_exception
    raise exc
  File "/home/medet/venv/lib/python3.8/site-packages/rest_framework/views.py", line 506, in dispatch
    response = handler(request, *args, **kwargs)
  File "/home/medet/elmedi/contract_management/api/views.py", line 36, in get
    contract_customer = ContractCustomer.objects.get(number=card_number)
  File "/home/medet/venv/lib/python3.8/site-packages/django/db/models/manager.py", line 85, in manager_method
    return getattr(self.get_queryset(), name)(*args, **kwargs)
  File "/home/medet/venv/lib/python3.8/site-packages/django/db/models/query.py", line 496, in get
    raise self.model.DoesNotExist(
contract_management.models.ContractCustomer.DoesNotExist: ContractCustomer matching query does not exist.
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /img/bg.jpg
Not Found: /
[2023-06-01 23:30:14 +0000] [760] [INFO] Starting gunicorn 20.1.0
[2023-06-01 23:30:14 +0000] [760] [INFO] Listening at: http://127.0.0.1:8000 (760)
[2023-06-01 23:30:14 +0000] [760] [INFO] Using worker: sync
[2023-06-01 23:30:14 +0000] [816] [INFO] Booting worker with pid: 816
[2023-06-01 23:30:14 +0000] [822] [INFO] Booting worker with pid: 822
[2023-06-01 23:30:14 +0000] [830] [INFO] Booting worker with pid: 830
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Unauthorized: /api/document_management/attached/document/
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /img/bg.jpg
/home/medet/venv/lib/python3.8/site-packages/django/views/generic/list.py:91: UnorderedObjectListWarning: Pagination may yield inconsistent results with an unordered object_list: <class 'package_service_management.models.Package'> QuerySet.
  return self.paginator_class(
/home/medet/venv/lib/python3.8/site-packages/django/views/generic/list.py:91: UnorderedObjectListWarning: Pagination may yield inconsistent results with an unordered object_list: <class 'package_icd_management.models.Package'> QuerySet.
  return self.paginator_class(
/home/medet/venv/lib/python3.8/site-packages/django/views/generic/list.py:91: UnorderedObjectListWarning: Pagination may yield inconsistent results with an unordered object_list: <class 'program_management.models.Package'> QuerySet.
  return self.paginator_class(
Not Found: /img/bg.jpg
Not Found: /img/bg.jpg
/home/medet/venv/lib/python3.8/site-packages/django/views/generic/list.py:91: UnorderedObjectListWarning: Pagination may yield inconsistent results with an unordered object_list: <class 'contract_management.models.ContractHospital'> QuerySet.
  return self.paginator_class(
/home/medet/venv/lib/python3.8/site-packages/django/views/generic/list.py:91: UnorderedObjectListWarning: Pagination may yield inconsistent results with an unordered object_list: <class 'invoice_management.models.Invoice'> QuerySet.
  return self.paginator_class(
Not Found: /img/bg.jpg
Not Found: /
Not Found: /
/home/medet/venv/lib/python3.8/site-packages/django/views/generic/list.py:91: UnorderedObjectListWarning: Pagination may yield inconsistent results with an unordered object_list: <class 'package_service_management.models.Package'> QuerySet.
  return self.paginator_class(
/home/medet/venv/lib/python3.8/site-packages/django/views/generic/list.py:91: UnorderedObjectListWarning: Pagination may yield inconsistent results with an unordered object_list: <class 'program_management.models.Package'> QuerySet.
  return self.paginator_class(
/home/medet/venv/lib/python3.8/site-packages/django/views/generic/list.py:91: UnorderedObjectListWarning: Pagination may yield inconsistent results with an unordered object_list: <class 'invoice_management.models.Invoice'> QuerySet.
  return self.paginator_class(
Not Found: /img/bg.jpg
/home/medet/venv/lib/python3.8/site-packages/django/views/generic/list.py:91: UnorderedObjectListWarning: Pagination may yield inconsistent results with an unordered object_list: <class 'directory.models.Insurance'> QuerySet.
  return self.paginator_class(
Not Found: /img/bg.jpg
/home/medet/venv/lib/python3.8/site-packages/django/views/generic/list.py:91: UnorderedObjectListWarning: Pagination may yield inconsistent results with an unordered object_list: <class 'directory.models.Hospital'> QuerySet.
  return self.paginator_class(
Not Found: /img/bg.jpg
/home/medet/venv/lib/python3.8/site-packages/django/views/generic/list.py:91: UnorderedObjectListWarning: Pagination may yield inconsistent results with an unordered object_list: <class 'contract_management.models.Contract'> QuerySet.
  return self.paginator_class(
/home/medet/venv/lib/python3.8/site-packages/django/views/generic/list.py:91: UnorderedObjectListWarning: Pagination may yield inconsistent results with an unordered object_list: <class 'contract_management.models.ContractCustomer'> QuerySet.
  return self.paginator_class(
/home/medet/venv/lib/python3.8/site-packages/django/views/generic/list.py:91: UnorderedObjectListWarning: Pagination may yield inconsistent results with an unordered object_list: <class 'invoice_management.models.Invoice'> QuerySet.
  return self.paginator_class(
/home/medet/venv/lib/python3.8/site-packages/django/views/generic/list.py:91: UnorderedObjectListWarning: Pagination may yield inconsistent results with an unordered object_list: <class 'contract_management.models.ContractHospital'> QuerySet.
  return self.paginator_class(
/home/medet/venv/lib/python3.8/site-packages/django/views/generic/list.py:91: UnorderedObjectListWarning: Pagination may yield inconsistent results with an unordered object_list: <class 'customer.models.Customer'> QuerySet.
  return self.paginator_class(
Not Found: /favicon.ico
Not Found: /img/bg.jpg
/home/medet/venv/lib/python3.8/site-packages/django/views/generic/list.py:91: UnorderedObjectListWarning: Pagination may yield inconsistent results with an unordered object_list: <class 'package_service_management.models.Package'> QuerySet.
  return self.paginator_class(
/home/medet/venv/lib/python3.8/site-packages/django/views/generic/list.py:91: UnorderedObjectListWarning: Pagination may yield inconsistent results with an unordered object_list: <class 'package_icd_management.models.Package'> QuerySet.
  return self.paginator_class(
Not Found: /favicon.ico
Not Found: /img/bg.jpg
Not Found: /favicon.ico
Not Found: /img/bg.jpg
Not Found: /img/bg.jpg
Not Found: /img/bg.jpg
/home/medet/venv/lib/python3.8/site-packages/django/views/generic/list.py:91: UnorderedObjectListWarning: Pagination may yield inconsistent results with an unordered object_list: <class 'contract_management.models.Contract'> QuerySet.
  return self.paginator_class(
/home/medet/venv/lib/python3.8/site-packages/django/views/generic/list.py:91: UnorderedObjectListWarning: Pagination may yield inconsistent results with an unordered object_list: <class 'package_icd_management.models.Package'> QuerySet.
  return self.paginator_class(
Not Found: /img/bg.jpg
Not Found: /img/bg.jpg
Not Found: /api/contract_management/contract_customer/07080808/detail
/home/medet/venv/lib/python3.8/site-packages/django/views/generic/list.py:91: UnorderedObjectListWarning: Pagination may yield inconsistent results with an unordered object_list: <class 'directory.models.Hospital'> QuerySet.
  return self.paginator_class(
Not Found: /img/bg.jpg
Not Found: /img/bg.jpg
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
/home/medet/venv/lib/python3.8/site-packages/django/views/generic/list.py:91: UnorderedObjectListWarning: Pagination may yield inconsistent results with an unordered object_list: <class 'program_management.models.Package'> QuerySet.
  return self.paginator_class(
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /cgi
Not Found: /
Not Found: /cgi
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /cgi
Not Found: /cgi
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /cgi
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /img/bg.jpg
Not Found: /img/bg.jpg
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /api/contract_management/contract_customer/787878/detail
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /img/bg.jpg
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
Not Found: /
